source/features/releases-tab.tsx

```diff
@@ -59,7 +59,7 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	await elementReady('.pagehead + *'); // Wait for the tab bar to be loaded
+	await elementReady('.pagehead + *, .js-repo-nav + *'); // Wait for the tab bar to be loaded
```

Comment:
Nothing exists after `.js-repo-nav`, but this should work
```suggestion
	await elementReady('.pagehead + *, .UnderlineNav-body + *'); // Wait for the tab bar to be loaded
```